### ISSUE TITLE
Zero Warnings

### ISSUE THREAD

**Issue by IanMayo** [serge-web/serge#4]
## User story
As a _maintainer_ I wish to _learn about potential bugs in new code_ by _examining lint/compiler warnings_

## Overview
We're inheriting a fairly large body of code.  It includes a number of react compiler warnings.  The presence of these masks the introduction of new warnings.

So, we should fix existing compiler warnings.

## Testing
The Serge client `yarn start` runs without reporting any warnings.

**Comment by IanMayo** [comment]
@heridehero, if you have chance this week, it would be great if you could start working through this task.

@foxleigh81 said:
> it would be great if [the team] could start working on fixing the linting errors on the site. They can see the output of each packages linting issues by typing yarn lint in that packages folder.

**Comment by zukilover** [comment]
I'm working on it

**Comment by zukilover** [comment]
@IanMayo @foxleigh81 Some of the linting fixes were included in [another PR](https://github.com/serge-web/serge/pull/82/files) I've made few days ago in serge-web/serge. How would you proceed with the merge, before I carry out with the rest of the fixes in this monorepo?

**Comment by IanMayo** [comment]
@zukilover - we spoke in Slack. You agreed to try to port the changes to `serge-web`, and if there aren't any changes that clash with the new monorepo architecture, we'll merge them into serge-web.

**Comment by zukilover** [comment]
@foxleigh81 @IanMayo I've opened a PR, but I need to retest in on my local. I got `Access to fetch at 'https://serge-dev.herokuapp.com/db/serge_info/' from origin 'http://localhost:3000' has been blocked by CORS policy` error when running on my machine. is there any workaround?

**Comment by IanMayo** [comment]
Hello @zukilover 

Yes, you have to ensure the server `REACT_SERVER_PATH` (in the `.env` file in `client\src`) matches where the app is being hosted from.

So, to run it on your local hosting, you would need to change it to `REACT_APP_SERVER_PATH='http://localhost:8080/'`

Yes, we should improve this.  By default, the app should use the base of it's URL as that path, IMHO.
